### DISCUSSION TITLE
Resize v2.5 Terminals and Split Windows?

### DISCUSSION THREAD

**Discussion by guzman109** [NvChad/NvChad#2757] [Q&A] [answered]
Does anyone know how to resize the terminals to be bigger with this new version? I used to have it in my plugins using nvterm but it doesn't seem to work now. 

Also, can you resize splits with key bindings? I couldn't find it but I may be looking in the wrong place.

**Comment by siduck** [comment, marked as answer]
https://nvchad.com/docs/config/nvchad_ui/#term
